test/apify/storages/request_queue_reset.ts

```diff
@@ -0,0 +1,32 @@
+// $ npm run build && ts-node test/apify/storages/request_queue_reset.ts
```

Comment:
this one should be an e2e test, so please move it to `test/e2e/test-request-queue`

(we might want to revisit the naming, i dont like the `test-` prefix, but lets be consistent for now)